src/etos_lib/logging/logger.py

```diff
@@ -189,7 +189,7 @@ def setup_otel_logging(
 def setup_logging(
     application: str,
     version: str,
-    environment: str,
+    environment: str,  # pylint: disable=unused-argument # This is kept to maintain backward compatibility
```

Comment:
Perhaps I could add a default None value also. That would allow for smoother future transitions to not using this at all.